app/models/ability.rb

```diff
@@ -0,0 +1,2 @@
+class Ability < Decidim::Ability
```

Comment:
@josepjaume it breaks in development when I remove this file, but I don't know why

```
ActionView::Template::Error (uninitialized constant Ability):
    1: <li><%= link_to t(".profile"), decidim.account_path %></li>
    2: <% if can? :read, :admin_dashboard %>
    3:   <li><%= link_to t(".admin_dashboard"), decidim_admin.root_path %></li>
    4: <% end %>
    5: <li><%= link_to t(".sign_out"), decidim.destroy_user_session_path, method: :delete, class: "sign-out-link" %></li>

cancancan (1.16.0) lib/cancan/controller_additions.rb:371:in `current_ability'
cancancan (1.16.0) lib/cancan/controller_additions.rb:394:in `can?'
actionpack (5.0.2) lib/abstract_controller/helpers.rb:68:in `can?'
/Users/amaia/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/bundler/gems/decidim-c400beaf7655/decidim-core/app/views/layouts/decidim/_user_menu.html.erb:2:in `___sers_amaia__rbenv_versions_______lib_ruby_gems_______bundler_gems_decidim_c___beaf_____decidim_core_app_views_layouts_decidim__user_menu_html_erb__3410750993160859078_70276087318260'
```